#Scenes/TestingScene.tscn

```diff
@@ -11,13 +11,47 @@
 [ext_resource type="Resource" uid="uid://ctx8jsvac84so" path="res://Cards/Resource/Card_Poison.tres" id="8_x6t2k"]
 [ext_resource type="Resource" uid="uid://d12g33rc6c3u5" path="res://Cards/Resource/Card_Heal.tres" id="9_ojxic"]
 [ext_resource type="Script" path="res://UI/DiscardPileUISetter.gd" id="10_pqly7"]
+[ext_resource type="Script" path="res://Cards/Card_DamageHealth.gd" id="10_qowiy"]
 [ext_resource type="Resource" uid="uid://3s4aet1ciesh" path="res://Cards/Resource/Card_damage_and_poison.tres" id="10_w0xgm"]
 [ext_resource type="Resource" uid="uid://bsrdu33ukb1ym" path="res://Cards/Resource/Card_EnemyAttack.tres" id="11_3k5t2"]
 [ext_resource type="Texture2D" uid="uid://caemucaya30wh" path="res://Cards/draw_pile.png" id="11_pw70x"]
 [ext_resource type="Texture2D" uid="uid://d4muqvs3etnr8" path="res://Cards/discard_pile.png" id="12_kxw48"]
 [ext_resource type="Script" path="res://UI/EndTurnButton.gd" id="14_dpe64"]
+[ext_resource type="Texture2D" uid="uid://hqkt8t1v2f5h" path="res://Cards/deck_pile.png" id="18_b54hn"]
+[ext_resource type="Script" path="res://UI/DeckPileUISetter.gd" id="19_ucc6f"]
+
+[sub_resource type="Resource" id="Resource_82ci8"]
+script = ExtResource("10_qowiy")
+damage_to_apply_to_target = 0.0
+damage_to_apply_to_caster = 0.0
+status_to_apply_to_target = Array[Resource("res://Status/StatusBase.gd")]([])
+status_to_apply_to_caster = Array[Resource("res://Status/StatusBase.gd")]([])
+affect_all_targets = false
+affect_all_casters = false
```

Comment:
These fields have been removed in #37 I believe. Should we wait for that PR to be merged in first? @Turtyo

Reply:
ah that's what you were talking about ! It's fine, we can merge this in and then #37 that will prevent having to merge the main into this PR again
If it's just something like that and not Paper using explicitly something deleted then it's fine

Reply:
Roger that, I will resolve.